dts/bindings/led/awinic,aw20216s.yaml

```diff
@@ -0,0 +1,28 @@
+description: Awinic AW20216S LED Driver
+
+compatible: "awinic,aw20216s"
+
+include: ["spi-device.yaml", "led-controller.yaml"]
+
+on-bus: spi
+
+properties:
+  en-gpios:
+    type: phandle-array
+    description: Enable pin
+  sync-gpios:
+    type: phandle-array
+    description: Sync pin
+  current-limit:
+    type: int
+    default: 150
```

Comment:
please, use SI units and appropriate suffixes. Also, no magic defaults, they need to be justified properly

ref https://github.com/devicetree-org/dt-schema/blob/main/dtschema/schemas/property-units.yaml

Reply:
Ya but this is unitless in that case. So maybe something like https://github.com/zephyrproject-rtos/zephyr/blob/main/dts/bindings/led/issi%2Cis31fl3733.yaml#L17-L25

Reply:
shouldn't that one use SI units, provided RI_SET is known?